Idno/Data/Mongo.php

```diff
@@ -0,0 +1,407 @@
+<?php
+
+/**
+ * MongoDB back-end for Known data.
+ *
+ * This is a wrapper for DataConcierge, but begins to move mongo specific settings
+ * to its own class.
+ * 
+ * @package idno
+ * @subpackage data
+ */
+
+namespace Idno\Data {
+
+    use Idno\Core\Idno;
+
+    class Mongo extends \Idno\Core\DataConcierge {
```

Comment:
are you asking if this should be implements instead of extends? DataConcierge is still a class, not an interface, so extends is correct.